### PR TITLE
Consolidate samba server container build

### PR DESCRIPTION
**Pull request by phlogistonjohn**
Depends on: #57 

The first set of changes merges the previous fedora and nightly files into a single container file.

Remaining changes clean up a few old things.

### CODE REVIEW

**Review by anoopcs9 — COMMENTED** [review]
Looks fine. But what about [smb.import.conf](https://github.com/samba-in-kubernetes/samba-container/blob/master/images/server/smb.import.conf)? Are we going to keep it under images/server?

**Comment by phlogistonjohn** [comment]
> Looks fine. But what about [smb.import.conf](https://github.com/samba-in-kubernetes/samba-container/blob/master/images/server/smb.import.conf)? Are we going to keep it under images/server?

Looks like it is unused too. We can remove it.

File: images/server/install-packages.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -ex
+
+get_custom_repo() {
+    url="$1"
+    fname="$(basename "$url")"
+    dest="/etc/yum.repos.d/${fname}"
+    dnf install --setopt=install_weak_deps=False curl
```
Comment (phlogistonjohn) [inline images/server/install-packages.sh:9]: missing a `-y` here

**Review by raghavendra-talur — APPROVED** [review]

**Review by anoopcs9 — APPROVED** [review]
lgtm, thanks.

**Pull request merged**